dashboard/src/components/ChapterCard.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <Card>
+        <template  #actions-left>
+            <FossClubLogo v-if="props.chapter.chapter_type == 'FOSS Club'" class="w-7 h-7 "></FossClubLogo>
+            <div v-else class="w-fit font-fff text-xs bg-gray-900 text-white p-2 rounded-sm flex text-center">{{ props.chapter.chapter_type.toUpperCase() }}</div>
+        </template>
+        <div class="flex justify-between items-baseline">
+            <div class="text-lg font-medium">{{ props.chapter.chapter_name }}</div>
+            <Button
+                :variant="'ghost'"
+                :size="'sm'"
+                label="Manage"
+                iconRight="chevron-right"
+                @click="() => $router.push(`/chapter/${encodeURI(props.chapter.name)}`)"
+            >
```

Comment:
Button component also takes a route, which it better since it will render as like IIRC

Reply:
changed